releasenotes/notes/otel-mapping-go-v0.5.2-bf02c3e60a.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Update the return value of MapMetrics, renames its field LanguageTags to Languages, and removes field HasRuntimeMetrics.
```

Comment:
We don't usually list library-only changes in the changelog, I think we can remove this unless it has user-facing consequences

Reply:
Makes sense to me, removed it